src/Staged/GHC/Generics/Types.hs

```diff
@@ -141,7 +62,24 @@ class Generic (a :: Type) where
     type Rep a = Translate (GHC.Rep a)
 
     to   :: Quote q => Rep a (Code q) x -> Code q a
+    default to :: (Rep a ~ Translate (GHC.Rep a), GHC.Generic a, GGeneric (Rep a), Quote q)
+               => Rep a (Code q) x -> Code q a
+    to = genericTo
+
     from :: Quote q => Code q a -> (Rep a (Code q) x -> Code q r) -> Code q r
+    default from :: (Rep a ~ Translate (GHC.Rep a), GHC.Generic a, GGeneric (Rep a), Quote q)
+      => Code q a -> (Rep a (Code q) x -> Code q r) -> Code q r
+    from = genericFrom
+
+-- | Use GHC generics to implement the 'to' method.
+genericTo :: (Rep a ~ Translate (GHC.Rep a), GHC.Generic a, GGeneric (Rep a), Quote q)
+  => Rep a (Code q) x -> Code q a
+genericTo = ghcGenericTo
```

Comment:
Should this just re-export the function from `GHCish` with the unconstrained type? As it is, it's likely not useful to export.

Reply:
I just did it in a separate commit.